### ISSUE TITLE
Error `dbus: Connection Refused`

### ISSUE THREAD

**Issue by morris821028** [morris821028/troubleshooting#4]
## Message ##
```
WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-9wOmF5UQ0y: Connection refused
```

## Solution ##
```
setenv NO_AT_BRIDGE 1
```
